Jenkins.master

```diff
@@ -0,0 +1,76 @@
+pipeline {
+    agent {
+        label 'docker01'
+    }
+    options {
+        ansiColor colorMapName: 'XTerm'
+    }
+    parameters {
+        string(
+            name: 'app_servers',
```

Comment:
I know that I used lowercase params names too. But they use uppercase in oficial Jenkins guidelines. 

https://jenkins.io/doc/book/pipeline/syntax/#parameters